tools/provision.py

```diff
@@ -207,6 +207,7 @@ def main(options):
     run(["sudo", "chown", "%s:%s" % (user_id, user_id), EMOJI_CACHE_PATH])
     run(["tools/setup/emoji/build_emoji"])
     run(["scripts/setup/generate_secrets.py", "--development"])
+    run(["tools/update-authors-json", "--use-fixture"])
```

Comment:
We'll want/need to either move this to a setUp() method for the test class add this to the manual setup instructions for the development environment.